File: modul 2/src/kegiatan/kegiatan1.py
from typing import List


def main() -> None:
    hewan: List[str] = []

    hewan.insert(0, "Angsa")
    hewan.insert(1, "Bebek")
    hewan.insert(2, "Cicak")
    hewan.insert(3, "Domba")
    hewan.insert(4, "Elang")
    hewan.insert(5, "Gajah")
    hewan.insert(6, "Badak")
    hewan.insert(7, "Bebek")

    print("\nOutput 1,2 :")
    print(hewan)
    print("Bebek = " + str(hewan.index("Bebek")) + "," +
          str(len(hewan) - hewan[::-1].index("Bebek") - 1))

    hewan.pop(1)
    print("\nOutput 3 :\n" + str(hewan))

    print("\nOutput 4 :")
    print("Index ke-0 : " + hewan[0])
    print("Index ke-2 : " + hewan[2])
    hewan.pop(0)
    print(hewan)

    print("\nOutput 5 :")
    hewan[0] = "Ular"
    print(hewan)
    hewan.insert(2, "Itik")
    print(hewan)

    print("\nOutput 6 :")
    hewan[2:5] = []
    print(hewan)

    print("\nOutput 7 :")
    print("Elemen pertama : " + hewan[0])
    print("Elemen Terakhir : " + hewan[-1])

    print("\nOutput 8,9 :")
    print("Jumlah index : " + str(len(hewan)))
    print("Posisi index Badak : " + str(hewan.index("Badak")))


if __name__ == "__main__":
    main()

File: modul 2/src/kegiatan/kegiatan1.java
package kegiatan;

import java.util.ArrayList;

public class kegiatan1 {

    public static void main(String[] args) {
        ArrayList<String> hewan = new ArrayList<>();

        hewan.add(0, "Angsa");
        hewan.add(1, "Bebek");
        hewan.add(2, "Cicak");
        hewan.add(3, "Domba");
        hewan.add(4, "Elang");
        hewan.add(5, "Gajah");
        hewan.add(6, "Badak");
        hewan.add(7, "Bebek");

        System.out.println("\nOutput 1,2 :");
        System.out.println(hewan);
        System.out.println("Bebek = " + hewan.indexOf("Bebek") + "," + hewan.lastIndexOf("Bebek"));

        hewan.remove(1);
        System.out.println("\nOutput 3 :\n" + hewan);

        System.out.println("\nOutput 4 :");
        System.out.println("Index ke-0 : " + hewan.get(0));
        System.out.println("Index ke-2 : " + hewan.get(2));
        hewan.remove(0);
        System.out.println(hewan);

        System.out.println("\nOutput 5 :");
        hewan.set(0, "Ular");
        System.out.println(hewan);
        hewan.add(2, "Itik");
        System.out.println(hewan);

        System.out.println("\nOutput 6 :");
        hewan.subList(2, 5).clear();
        System.out.println(hewan);

        System.out.println("\nOutput 7 :");
        System.out.println("Elemen pertama : " + hewan.get(0));
        System.out.println("Elemen Terakhir : " + hewan.get(hewan.size() - 1));

        System.out.println("\nOutput 8,9 :");
        System.out.println("Jumlah index : " + hewan.size());
        System.out.println("Posisi index Badak : " + hewan.indexOf("Badak"));

    }
}
